Changelogs for Digital Twin Manager Plugins 2.1.1 (#3106)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1021-2-1-fix-for-root-asset-movement-and-duplicate-name-validation-in-asset-navigator.md

---
date: ""

> title: "Fix for root asset movement and duplicate name validation in Asset Navigator"

Reviewer: ```suggestion
title: "Support for moving root assets and duplicate name validation in Asset Navigator"
```

> title: "Support for moving root assets and duplicate name validation in Asset Navigator"

Reviewer 2: ```suggestion
title: "Added support for moving root assets and duplicate name validation in the asset navigator"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-1938"
version: "1021.2.1"
---

> Root-level assets can now be moved under groups created in Cockpit. Additionally, validation has been added to prevent moving assets or groups to locations where assets or groups with identical names already exist, ensuring consistency and avoiding naming conflicts.

Reviewer 2: ```suggestion
In the asset navigator, root-level assets can now be moved under groups created in the Cockpit application. Additionally, validation has been added to prevent moving assets or groups to locations where assets or groups with identical names already exist, ensuring consistency and avoiding naming conflicts.
```

> Previously, root-level assets could not be moved under groups created in Cockpit. This has been resolved, allowing such movements. Additionally, validation has been introduced to prevent moving assets or groups into a location where an asset or group with the same name already exists, ensuring consistency and avoiding naming conflicts.

Reviewer: ```suggestion
Root-level assets can now be moved under groups created in Cockpit. Additionally, validation has been added to prevent moving assets or groups to locations where assets or groups with identical names already exist, ensuring consistency and avoiding naming conflicts.
```
